public/js/monitor.js

```diff
@@ -523,17 +582,8 @@ function vpnBannerLogic() {
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
 
-  const preferredLanguages = navigator.languages;
-  const preferredFirstLanguageIsTier1 = checkIfTier1(preferredLanguages[0]);
-
-  if (!preferredFirstLanguageIsTier1) {
-    return;
-  }
-
-  // Only show banner if users first language is English, Germand or French variant
-  if (["en", "de", "fr"].some(lang=>preferredLanguages[0].includes(lang))) {
-    vpnBannerLogic();
-  }
+  // Show vpn-banner for locales not in the blocked list
+  if (!["zh-CN"].some(lang => navigator.languages[0].includes(lang))) initVpnBanner();
```

Comment:
Should the blocked list be in the AppConstants file or something?

Reply:
Good suggestion – I added this, and also updated the recent email promo addition to use the same var/logic.